config/environments/production.rb

```diff
@@ -47,15 +47,15 @@
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # Local changes
-  config.force_ssl = true
+  config.force_ssl = ENV['RAILS_PROD_NOSSL'].blank?
 
   # Handle STS here instead of Apache, or Rails duplicates header contents
   # Also unset cache-control header in HTTPS vhost for same reason
-  config.ssl_options = { hsts: { preload: true } }
+  #config.ssl_options = { hsts: { preload: true } }
```

Comment:
This should stay uncommented. I'll test this locally and push a fix before I merge.

Reply:
Works as intended in local testing :+1: